### PR TITLE
feat(staking): tx modal modification

### LINKED ISSUE #13594: Duplicated minimum amount to stake ETH in codebase
We are using two variables for minimum staking amount `0.1`: `MIN_ETH_AMOUNT_FOR_STAKING` and `MIN_AMOUNT` in `stake` method in `stake.ts` file.

Let's unify it to use just one `MIN_ETH_AMOUNT_FOR_STAKING`.

Also modify error 
```
throw new Error(`Min Amount ${MIN_AMOUNT} wei`); 
``` 
to throw it in `ETH` not in `wei`

### LINKED ISSUE #13600: Remove network send warning when staking/unstaking/claiming
Please do not show this warning in transaction review modal when staking/unstaking/claiming

Image

### LINKED ISSUE #13601: Hide broadcast option for staking/unstaking/claiming in review modal
There is no option to turn on/off the broadcast of stake/unstake/claim, so let's hide it from review tx modal
- also for bump

Image

### PR DESCRIPTION
**Pull request by dev-pvl**
## Description

This PR introduces three key improvements:

1. Unified minimum amount constants 
2. Removed the network send warning
3. Hid broadcast option

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/13601 
Resolve https://github.com/trezor/trezor-suite/issues/13600 
Resolve https://github.com/trezor/trezor-suite/issues/13594

## Screenshots:
tx_modal_mod

### CODE REVIEW

File: packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx [hunk 1]
```diff
@@ -168,7 +169,9 @@ export const TransactionReviewModalContent = ({
                 setIsSending={() => setIsSending(true)}
                 ethereumStakeType={ethereumStakeType || undefined}
             />
-            <TransactionReviewEvmExplanation account={selectedAccount.account} />
+            {!ethereumStakeType && (
+                <TransactionReviewEvmExplanation account={selectedAccount.account} />
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx]: Could you do this check inside TransactionReviewEvmExplanation component please?

**Review by tomasklim — CHANGES_REQUESTED** [review]
Would be great to have at least 1 commit per issue

Not all issues were linked, so I edited first post

File: packages/suite/src/utils/suite/stake.ts [hunk 2]
```diff
@@ -56,10 +57,10 @@ export const stake = async ({
 }: StakeTxBaseArgs & {
     amount: string;
 }) => {
-    const MIN_AMOUNT = new BigNumber('100000000000000000');
     const amountWei = toWei(amount, 'ether');
 
-    if (new BigNumber(amountWei).lt(MIN_AMOUNT)) throw new Error(`Min Amount ${MIN_AMOUNT} wei`);
+    if (new BigNumber(amount).lt(MIN_ETH_AMOUNT_FOR_STAKING))
+        throw new Error(`Min Amount ${MIN_ETH_AMOUNT_FOR_STAKING} ETH`);
```
Comment (tomasklim) [inline packages/suite/src/utils/suite/stake.ts]:
```suggestion
    if (new BigNumber(amount).lt(MIN_ETH_AMOUNT_FOR_STAKING)) {
        throw new Error(`Min Amount ${MIN_ETH_AMOUNT_FOR_STAKING} ${symbol.toUpperCase()}`);
    }
```

bracelets and symbol

File: packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx (hunk 1 above)
Comment (dev-pvl) [reply packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx]: sure, https://github.com/trezor/trezor-suite/pull/13732/commits/b3e350b47f6b661b602563fd6399b016e269e24f

File: packages/suite/src/utils/suite/stake.ts (hunk 2 above)
Comment (dev-pvl) [reply packages/suite/src/utils/suite/stake.ts]: done, https://github.com/trezor/trezor-suite/pull/13732/commits/b3e350b47f6b661b602563fd6399b016e269e24f

**Comment by dev-pvl** [comment]
These issues are small and I added them in one commit. I'll create a separate commit for every issue next time. Would that be fine for you?

**Review by tomasklim — APPROVED** [review]

**Pull request merged**